pkg/cmd/server/api/types.go

```diff
@@ -71,6 +71,14 @@ type NodeConfig struct {
 	KubeletArguments ExtendedArguments
 }
 
+// NodeNetworkConfig provides network options for the node
+type NodeNetworkConfig struct {
```

Comment:
hmm, ok... keep the type NodeNetworkConfig, but make the json param `networkConfig` so we don't stutter in the node config file